testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -120,7 +120,8 @@ class DensityParameters(object):
     topology = TPR
     trajectory = XTC
     delta = 2.0
-    selections = {'static': "name OW",
+    selections = {'none': "byres resname LIG and around 8 protein", #this selection yields nothing
```

Comment:
Not the best way to choose an empty selection. Although it is tagged as `none` the `byres resname LIG and around 8 protein` sounds plausible. I would do something that is more obvious like `None` or `"resname None"`